lib/content/routes.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * @fileoverview Content system routing functions
+ *
+ */
+const config = require('config');
+const types = require('punchcard-content-types');
+const _ = require('lodash');
+
+const database = require('../database');
+const utils = require('../utils');
+
+/**
+ * All Content-Types Landing page
+ * @param {object} res - HTTP Response
+ * @param {object} allTypes all content types available to the CMS
+ */
+const all = (res, allTypes) => {
+  res.render('content/home', {
```

Comment:
The other functions in this file all `return res.render` from a promise. Why isn't this being returned? Should we continue our practice from init of using promises as a shared interface even if they aren't necessarily required and turn this in to a promise?